data/index.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "author": "Abraham Lincoln",
+    "text": "A house divided against itself cannot stand.",
+    "tags": [
+      "Abraham",
+      "Lincoln"
+    ]
+  },
+  {
+    "author": "Albert Einstein",
+    "text": "If A is success in life, then A equals x plus y plus z. Work is x; y is play; and z is keeping your mouth shut."
+  },
+  {
+    "author": "Alexis Carrel",
+    "text": "All great men are gifted with intuition. They know without reasoning or analysis, what they need to know."
+  },
+  {
+    "author": "Amy Tan",
+    "text": "I am like a falling star who has finally found her place next to another in a lovely constellation, where we will sparkle in the heavens forever."
+  },
+  {
+    "author": "Lao Tzu",
+    "text": "Silence is a source of great strength."
+  },
+  {
+    "author": "Oscar Wilde",
+    "text": "Be yourself; everyone else is already taken.",
+    "id": "_3tkkf2jmd",
+    "isDeleted": false,
+    "createdAt": 1618988798267,
+    "updatedAt": 1619035567993
+  },
+  {
+    "author": "Marilyn Monroe",
+    "text": "I'm selfish, impatient and a little insecure. I make mistakes, I am out of control and at times hard to handle. But if you can't handle me at my worst, then you sure as hell don't deserve me at my best.",
+    "id": "_lpwgs2oyi",
+    "isDeleted": false,
```

Comment:
What is the purpose of `isDeleted`? I don't see a necessity of such sort of data even for the case with mocked data.